app/views/saved_scenarios/show.html.haml

```diff
@@ -1,34 +1,52 @@
-.scenario_wrapper#show_scenario
-
+%main.saved-scenario__main
   - if @warning
     .warning
       = @warning
 
-  %h1= (@scenario.title or t 'scenario.no_title' ).truncate(40, separator: ' ')
+  .saved-scenario__title
+    %h1= (@scenario.title or t 'scenario.no_title' ).capitalize
+
+  .saved-scenario__content
+    .saved-scenario__description
+      .saved-scenario__subhead
+        = link_to "",
+                  load_saved_scenario_path(@saved_scenario),
+                  class: "saved-scenario__nav tooltip fa fa-folder-open-o",
+                  title: t('saved_scenario.tooltips.open')
+
+        = link_to "", factsheet_scenario_path(@saved_scenario.scenario_id),
+                  class: "saved-scenario__nav tooltip fa fa-bar-chart",
+                  title: t('saved_scenario.tooltips.factsheet')
 
-  = link_to t("scenario.load"),
-            load_saved_scenario_path(@saved_scenario),
-            class: "button primary"
+        -# = link_to "",
+                  "#",
+                  class: "saved-scenario__nav tooltip fa fa-truck",
+                  title: t('saved_scenario.tooltips.transport_region')
 
-  = link_to t("scenario.load_factsheet"),
-            factsheet_scenario_path(@saved_scenario.scenario_id),
-            class: "button"
+    - if @scenario.description
+      .saved-scenario__field
+        %p.saved-scenario__field-label
+          = t('saved_scenario.description').capitalize
+        %p.saved-scenario__field-value= @description
 
-  - if @scenario.description
-    %em= @description
 
-  %ul
     - if @scenario.created_at
-      %li
-        %strong #{t('scenario.created')}:
-        = I18n.localize(@scenario.parsed_created_at, :format => :long)
-    %li
-      %strong #{t('scenario.area_code')}:
-      = t @scenario.area_code
-    %li
-      %strong #{t('scenario.end_year')}:
-      = @scenario.end_year
-
-  %p
-    %small{:style => "font-size: 12px; font-weight: normal"}
-      = t("scenario.overwritten")
+      .saved-scenario__field
```

Comment:
Since these are basically "label" => "value", I think [a definition list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) would be a better fit than `div`s and `p`aragraphs.